checks/evaluation/pinned_dependencies.go

```diff
@@ -289,27 +275,33 @@ func createReturnForIsGoInstallPinned(pr map[checker.DependencyUseType]pinnedRes
 ) (int, error) {
 	return createReturnValues(pr, checker.DependencyUseTypeGoCommand,
 		"go installs are pinned",
+		"no go installs found",
 		dl)
 }
 
 func createReturnValues(pr map[checker.DependencyUseType]pinnedResult,
-	t checker.DependencyUseType, infoMsg string,
+	t checker.DependencyUseType, maxResultMsg string, inconclusiveResultMsg string,
 	dl checker.DetailLogger,
 ) (int, error) {
-	// Note: we don't check if the entry exists,
-	// as it will have the default value which is handled in the switch statement.
+	// If there are zero dependencies of this type, it will generate an inconclusive score,
+	// so we can disconsider it in the aggregated score.
+	// If all dependencies of this type are pinned, it will get a maximum score.
+	// If 1 or more dependencies of this type are unpinned, it will get a minimum score.
 	//nolint
-	r, _ := pr[t]
-	switch r {
-	default:
-		return checker.InconclusiveResultScore, fmt.Errorf("%w: %v", errInvalidValue, r)
-	case pinned, pinnedUndefined:
+	r := pr[t]
+
+	switch r.total {
+	case 0:
+		dl.Info(&checker.LogMessage{
```

Comment:
let's make this a debug. we don't need a detail for every ecosystem a project isn't using